src/sentry/debug_files/debug_files.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import annotations
+
+from datetime import datetime, timedelta
+from typing import Sequence
+
+import pytz
+from django.db import router
+from django.db.models import Q
+
+from sentry import options
+from sentry.models.debugfile import ProjectDebugFile
+from sentry.utils import metrics
+from sentry.utils.db import atomic_transaction
+
+# TODO: merge this with artifact bundles
+# Number of days that determine whether a debug file is ready for being renewed.
+AVAILABLE_FOR_RENEWAL_DAYS = 30
+
+
+def maybe_renew_debug_files(query: Q, debug_files: Sequence[ProjectDebugFile] | None = None):
+    if options.get("debug-files.enable-renewal") == 1.0:
+        # NOTE: this metric also covers the early-return
+        with metrics.timer("debug_files_renewal"):
+            renew_debug_files(query, debug_files or list())
+
+
+def renew_debug_files(query: Q, debug_files: Sequence[ProjectDebugFile]):
+    # We take a snapshot in time that MUST be consistent across all updates.
+    now = datetime.now(tz=pytz.UTC)
```

Comment:
Could we use `timezone.now()` from `django.utils` instead? That wrapper saves you some imports, and is more consistent with timestamp generation elsewhere in the app.